src/main/resources/db/migration/V0.5.24__create-breeding_method-table.sql

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
Looks like you need to change the file name to V0.5**.25**__create-breeding_method-table.sql in order to get it to pass the checks.

Reply:
thx